pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -87,10 +87,10 @@ func NewPodNetworkAvalibilityInvariant(initializationInfo monitortestframework.M
 func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	openshiftTestsImagePullSpec, notSupportedReason, err := pna.getImagePullSpec(ctx, adminRESTConfig)
 	if err != nil {
-		return err
+		notSupportedReason = "Failed to find test image pullspec"
+		return nil
```

Comment:
Why does this assign to notSupportedReason, and then not use it? It this supposed to be pna.notSupportedReason?